Jenkinsfile

```diff
@@ -72,14 +72,15 @@ pipeline {
         stage('Build') {
             steps {
                 echo 'Building'
-                sh 'mvn -U -B -e clean install -DskipTests apache-rat:check'
+                sh 'mvn -U -B -e apache-rat:check'
+                sh 'mvn -U -B -e ${MVN_LOCAL_REPO_OPT} clean install -DskipTests'
             }
         }
 
         stage('Tests') {
             steps {
                 echo 'Running tests'
-                sh 'mvn verify -Prun-its'
+                sh 'mvn verify -Prun-its ${MVN_LOCAL_REPO_OPT} -Dinvoker.streamLogsOnFailures=true'
```

Comment:
Yes, because you don't want them on your PC, probably. You have build.log files there. That's what I find much more comfortable than scrolling through a console or even copy lots from there.